main/src/library/File.flix

```diff
@@ -1161,4 +1161,47 @@ namespace File {
                 Err(getMessage(ex))
         }
 
+    ///
+    /// Deletes the given file or directory `f`.
+    ///
+    /// If `f` is a directory it must be empty.
+    ///
+    pub def delete(f: String): Result[Unit, String] \ IO =
+        try {
+            import new java.io.File(String): ##java.io.File \ IO as newFile;
+            import java.io.File.toPath(): ##java.nio.file.Path \ IO;
+            import static java.nio.file.Files.delete(##java.nio.file.Path): Unit \ IO;
+
+            let javaFile = newFile(f);
+            let javaPath = toPath(javaFile);
+
+            Ok(delete(javaPath))
+        } catch {
+            case ex: ##java.io.IOException =>
```

Comment:
Should it be checked for [SecurityExceptions](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/SecurityException.html) too, since [Files.delete](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/Files.html#delete(java.nio.file.Path)) can throw this error? It does not seem to be a subclass of [IOException](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/IOException.html).

Reply:
Nah, thats okay